app/_hub/yesinteractive/kong-log-google/index.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Google Analytics Log
+publisher: Yes Interactive
+
+categories: # (required) Uncomment all that apply.
+  #- authentication
+  #- security
+  #- traffic-control
+  #- serverless
+  #- analytics-monitoring
+  #- transformations
+  - logging
+  #- deployment
+
+type: plugin
+
+desc: Log API transactions to Google Analytics
+description: This plugin logs your Kong gateway transactions to Google Analytics. Plugin is a modification of the Kong HTTP Log plugin.
+
+support_url: https://github.com/yesinteractive/kong-log-google/issues
+
+source_url: https://github.com/yesinteractive/kong-log-google
+
+license_type: MIT
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0-x
+      - 1.5-x
+      - 1.4-x
+      - 1.3-x
+  enterprise_edition:
+    compatible:
+      - 1.5-x
+      - 1.3-x
```

Comment:
Can you add a line break right before params? It looks like only 1.3-x is displaying in the preview, wondering if this is why.

Reply:
had two param sections.... removed the one not needed. should be good to go.